installer/charts/values.yaml.tpl

```diff
@@ -6,9 +6,10 @@
 {{- $gitops := required "GitOps settings" .Installer.Features.openShiftGitOps -}}
 {{- $pipelines := required "Pipelines settings" .Installer.Features.openShiftPipelines -}}
 {{- $quay := required "Quay settings" .Installer.Features.redHatQuay -}}
+{{- $quayEnabled := not (lookup "v1" "Secret" .Installer.Namespace "rhtap-nexus-integration") -}}
```

Comment:
That should be `nexusEnabled`. You need to add logic to support both `quay.Enabled` and/or `nexusEnabled`.

Reply:
IIUC we only need to worry about enable/disable quay but not nexus because we are not installing nexus operator. We only want the `rhtap-cli integrations` . Here with `quayEnabled`, I am disabling quay if `rhtap-nexus-integration` secrets are available but if not just enable quay to get it installed. Let me know if my understanding is not correct

Reply:
1. You would be missing an 'or' condition I believe. `quayEnabled` will be true if there is no nexus integration even if the user disabled quay in the config.yaml.
2. The logic you are putting in place is saying "you cannot have quay and nexus at the same time". Do we know that it is true? Is DH breaking when that is the case?